package.json

```diff
@@ -78,7 +78,7 @@
         "playwright": "^1.38.0",
         "source-map-support": "^0.5.21",
         "tslib": "^2.5.0",
-        "typescript": "^5.3.2",
+        "typescript": "^5.4.0-dev.20231206",
```

Comment:
Is *dev* ok here ?

Reply:
Yeah, it's okay. We previously had a dev version in #51787, for example.

I'm not totally sure about the `^`, though; not clear to me if this will be relocked every day or what (I'll test quick).